frontend/src/pages/AbsencePlanner.tsx

```diff
@@ -143,11 +143,53 @@ export const AbsencePlanner = () => {
   };
 
   const removeTimeEntries = async (entryIds: number[]) => {
-    let removed = undefined;
+    let allRemoved: boolean = true;
     for await (let entryId of entryIds) {
       let entry: TimeEntry = { id: entryId, hours: 0 };
-      removed = await reportTime(entry, onErrorRemovingEntries, context);
+      const result = await reportTime(entry, onErrorRemovingEntries, context);
+      if (!result) {
+        allRemoved = false;
+      }
+    }
+    return allRemoved;
+  };
+
+  const onUpdateAbsenceRanges = async (
+    oldEntryIds: number[],
+    newAbsenceStart: Date,
+    newAbsenceEnd: Date
+  ) => {
+    const removedAll = await removeTimeEntries(oldEntryIds);
+    if (removedAll) {
+      const reportable_days = getReportableDays(newAbsenceStart, newAbsenceEnd);
+      const added = await reportAbsenceTime(reportable_days);
+      if (added) {
```

Comment:
We have to check that the date does not alrerady exist. I could update two rows to the same date:
![image](https://user-images.githubusercontent.com/105041852/190214504-8cf9141c-f49a-44df-934c-d927e65a2a71.png)
So it added 16 hours to today's date.
Well, it is of course only to show that it gets updated but this is a check we will have to do later as well so better to implement the check now.